.github/workflows/CI.yml

```diff
@@ -18,5 +18,6 @@ jobs:
         yarn install
         yarn run build --if-present
         yarn test a --watchAll=false
+        yarn test:coverage
```

Comment:
can we merge these 2 lines in just 1? there is no point in running tests twice.

Reply:
Sure! I have just done it.